gspread-project/push_data.py

```diff
@@ -72,7 +79,7 @@ def run(user_email: str, file: str, service_file_path: str, sheet_title: str, wo
 if __name__=="__main__":
     user_email = "gbotemibolarinwa@gmail.com"
```

Comment:
Be sure to add comments throughtout your code so that future you and your teammates (should you have any) will understand the code easily.